applications/GeoMechanicsApplication/tests/cpp_tests/test_apply_constant_phreatic_multi_line_pressure_process.cpp

```diff
@@ -129,14 +129,15 @@ KRATOS_TEST_CASE_IN_SUITE(ApplyConstantPhreaticMultiLinePressureProcessDoesNotTh
                 "y_coordinates": [0.0, 1.0],
                 "z_coordinates": [0.0, 0.0],
                 "gravity_direction": 1,
-                "out_of_plane_direction": 2
+                "out_of_plane_direction": 2,
+                "table": [0, 0]
             }  )"};
 
     KRATOS_EXPECT_TRUE(CanCreateInstanceOfApplyConstantPhreaticMultiLinePressureProcessWithoutFailure(
         r_model_part, test_parameters))
 
     test_parameters.RemoveValue("x_coordinates");
-    test_parameters.AddVector("x_coordinates", ScalarVector{5, 1.0});
+    test_parameters.AddVector("x_coordinates", ScalarVector{2, 1.0});
```

Comment:
Does this mean that if we create a 'constant' multiline process, with e.g. 2 tables in the parameters, we cannot have 5 x_coordinates (while the tables aren't used)? Please correct me if I'm wrong

Reply:
There should be as many entries in the table list as there are points. However a entry in the table list may be 0 to indicate a constant value.